mods/ts/rules/defaults.yaml

```diff
@@ -382,6 +382,8 @@
 		ReferencePoint: Bottom, Right
 		RequiresCondition: hospitalheal
 	RevealOnFire:
+	RejectsOrders:
```

Comment:
How about adding `Except: Move`? That way at least Move orders are still possible, while anything potentially problematic - especially `Stop` - is not.